server/views/pages/accessibilityStatement.njk

```diff
@@ -0,0 +1,128 @@
+{% extends "layout.njk" %}
+
+{% set pageTitle = applicationName + " accessibility statement" %}
+{% set pageId = 'accessibility-statement-page' %}
+{% set cvlLink = "https://create-and-vary-a-licence.hmpps.service.justice.gov.uk" %}
```

Comment:
this should probably be `config.domain`, so it varies per environment - you could add it to `app.locals`?

Reply:
or pass it through the render 🤷‍♂️

Reply:
done